docker/php/Dockerfile

```diff
@@ -1,8 +1,11 @@
 FROM php:7.2.7-fpm-alpine3.7
 RUN apk update \
     && apk upgrade \
-    && apk add git bash msmtp freetype libpng libjpeg-turbo freetype-dev libpng-dev libjpeg-turbo-dev;
+    && apk add git bash msmtp freetype libpng libjpeg-turbo freetype-dev libpng-dev libjpeg-turbo-dev autoconf build-base;
 RUN docker-php-ext-install mysqli pdo_mysql
 RUN docker-php-ext-install gd
+RUN pecl install apfd
+RUN echo 'extension=apfd.so' > /usr/local/etc/php/conf.d/apfd.ini
+RUN mkdir -p /tmp/scss_cache/scss && chmod -R 0777 /tmp/scss_cache
```

Comment:
Confusing a few different changes for different reasons.

Is the scss_cache change still needed?

Reply:
I haven't had a chance to test for that.

Reply:
Also, the APFD change is something of a mixed bag that I still need to exercise a bit more as to whether I really need it for what I'm doing here. PHP has a bad defect that they refuse to correct, insisting that PUT parameters should always been in the URL (they should not be). However, the main place it really breaks is when paired with `FormData`, which I'd started to use in Javascript. Now, however, I'm mainly using JSON instead all the way. I need to test whether apfd is needed for that. If it isn't, this whole change can go away.

Reply:
Honestly JSON should be our standard.  It is not hard to convert FormData into JSON.

Reply:
Agreed. The spike PR is not currently up to date, but my spike branch already made this change (including a small update to `apiRequest` to fully support it).

Reply:
These changes appear not to be need for JSON payloads, so I'm withdrawing them unless something else comes along that seems to need them.